OpenTrons/opentrons-api
# fix(protocol-designer): replace inputfield with components's one

# Overview
This should be reviewed after the launch.
replace inputfield with components's one

close RQA-3731

### before
<img width="502" alt="Screenshot 2024-12-02 at 3 11 05 PM" src="https://github.com/user-attachments/assets/cb0f1098-b450-4fd1-9695-9b4d48441bdf">


### after
<img width="502" alt="Screenshot 2024-12-02 at 3 11 15 PM" src="https://github.com/user-attachments/assets/b9a10cae-b9cc-40d3-88ee-d81aacf2dc6e">



## Test Plan and Hands on Testing
1. create/import a protocol
2. click edit metadata modal
3. change items
check there is no issue like [this](https://opentrons.slack.com/archives/C07NH7NU4D6/p1733336968477589?thread_ts=1733325616.225699&cid=C07NH7NU4D6)


## Changelog



## Review requests



## Risk assessment
low
